createlinks-imapsync

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/perl -w
+
+#
+# Copyright (C) 2018 Nethesis S.r.l.
+# http://www.nethesis.it - support@nethesis.it
+# 
+# This script is part of NethServer.
+# 
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+# 
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+use strict;
+use esmith::Build::CreateLinks qw(:all);
+
+
+#
+# UPDATE event
+#
+
+my $event = 'nethserver-mail-imapsync-update';
+
+
+event_actions($event, qw(
+    initialize-default-databases 00
+));
+
+$event = 'imapsync-start';
+
+event_actions($event, qw(
+    nethserver-imapsync-systemdRun 10
```

Comment:
Please use lower case chars only.

Reply:
I love perl, mainly for this syntax, but I will change it. Agreed

Reply:
done